### PR TITLE
Add rough html outline

### PR DESCRIPTION
**Pull request by cagallo**
#### What's this PR do?

- Added rough html outline 
- Added font to CSS 

#### Where should the reviewer start?
 
- html

#### How should this be manually tested?

- N/A

#### Any background context you want to provide?

- section with cards with recipe cards with eventually be in the JS (lines 107-14)

#### What are the relevant tickets?

- html

#### Screenshots (if appropriate)

- N/A

#### Questions:

- None!

### CODE REVIEW

**Review by aliroemhildt — APPROVED** [review]
Everything looks great to me!

File: src/index.html [hunk 1]
```diff
@@ -1,10 +1,120 @@
...
+      <button>Search</button>
+    </div>
+    <div>
+      <button class="hidden">Home</button>
+      <button>Recipes to Cook</button>
+      <button>Favorite Recipes</button>
+    </div>
+  </nav>
+  <main>
+    <section class="filter">
```
Comment (anewb87) [inline src/index.html:25]: 19 filters, tedious but looks good.

File: src/index.html [hunk 2]
```diff
@@ -1,10 +1,120 @@
...
+        </div>
+        <div>
+          <input type="checkbox" name="starter">
+          <label for="starter">Starter</label>
+        </div>
+        <button>Apply Filter</button>
+      </div>
+    </section>
+    <section class="recipes">
+      <!-- HTML for each recipe card, this will be moved to JS  -->
```
Comment (anewb87) [inline src/index.html:108]: Appreciate the comment so we know what's to come.

**Review by anewb87 — COMMENTED** [review]
Looks good! Like what where we've gotten so far.

**Comment by kammon10** [comment]
looks great!

**Review by kammon10 — COMMENTED** [review]
looks great!!

**Pull request merged**